dictionaries/en_GB/src/README.md

```diff
@@ -1,3 +1,11 @@
 # Source Directory
 
-All source files used to generate the dictionary should be stored in this directory.
+All source files used to generate the `en_GB` dictionary should be stored in this directory.
+
+> [!WARNING]
+> The files in this folder are only about British English licensed under LGPL (a restrictive license)
+
+## Adding words
+
+> [!CAUTION]
+Because of the license of this dictionary contributions are discouraged, you should consider contributing to [`en_GB-MIT`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_GB-MIT/src/) dictionary instead.
```

Comment:
Please review this @Jason3S

Reply:
I'm ok with the `en_GB` license, I just cannot bundle it with `cspell` by default. People can and should install this version if that is what they want to use.

Note: Words added here are also included in the `en_GB-MIT` version. So there is no downside to adding them here.

Reply:
> I'm ok with the en_GB license, I just cannot bundle it with cspell by default. People can and should install this version if that is what they want to use.

Yes, they can. As long as they are getting the additional words from en_GB-MIT but not the opposite

Reply:
Another solution could be to add an en_GB_shared source

But it increases the complexity

Reply:
I think there is a bit of confusion here.

Only 3rd party sources are tainted by the LGPL license. We can choose to publish our own working under whatever license we want. The same set of words can be published under multiple licenses. All `additional_words` files are considered our own work, so the LGPL license of the published package doesn't count.

Reply:
It should be OK. I might open a PR to make things clearer about the license differences between the code, and the packaging, but it's another dragon to fight, so it would be another PR

Reply:
I opened
- #4408
- #4409
- #4410

To provide different way to try to solve the issue I see with the current LICENSE file